### DISCUSSION TITLE
Phantom Fails to sign transactions on Chrome with Invalid Account Discriminator

### DISCUSSION THREAD

**Discussion by jacksturt** [phantom/docs#89] [Help & Support] [answered]
### Summary

An error with the Phantom Extension on Chrome. For some users, they are unable to interact with our smart contract using the phantom extension on chrome. It always gives the error Invalid Account Discriminator. But, when they switch the same private key to the phantom extension on Brave, they can interact with the program no problem.

Example failed tx send from phantom 23.11.0 on Chrome: https://solscan.io/tx/4oiovNqSkWfn5GGafcmuxsDeks4ome59xmhgrhpc8ptRc2BAptFuApZbdq9Jnye8v2TxXdEmQymEe4nxJyyVmxPv

Same tx being sent from phantom 23.11.0 on Brave: 
https://solscan.io/tx/3V5QbDBFcr6LRgbN1FvMRQ5BksqK2RYj8gmK71RQ4XZFhv2bjLxpwrKTH6xgmPMUUaY6L6EnmVxu2AahWawdScBo

I was able to solve this issue once, but it is plaguing multiple users, preventing them from being paid out.

### Steps to Reproduce

This seems to only happen for some users. If you would like, I can connect you with them:
1. Connect to a dApp with phantom on chrome
2. Try to sign and send a transaction, and get the error `Invalid Account Discriminator`
3. Switch to brave, and try to send the same transaction with the same private key
4. See that this succeeds without an issue

### Phantom Version

23.11.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hey @jacksturt,

Same as previous request, would you please provide the dApp where you're seeing this behavior?

**Comment by jacksturt** [comment]
Hey, sorry for the late reply, we just launched on Tuesday so it's been very busy. The site where this is happening is https://pro.lancer.so. It does not happen for everybody, but one of our team members can reproduce the error consistently. I would be happy to set up a time with the three of us to debug this live.

**Reply by adamdelphantom** [reply]
Would also like to see if this issue persists with 23.12.0, please

**Comment by fororo12** [comment]
I need help I did app interaction, but it failed my one sol did not exchange to dollars and I cannot see my one sol too please help me
![Screenshot 2023-12-21 110509](https://github.com/phantom/docs/assets/118480182/bdf16c29-cbd7-4c7f-b125-e55ec42f3fc1)
![Screenshot 2023-12-21 110443](https://github.com/phantom/docs/assets/118480182/ed2c4f15-2c80-49e4-9436-2b246a001a0e)

**Reply by adamdelphantom** [reply, marked as answer]
Hello,


Please consider contacting Phantom Support regarding this. They can be reached in your wallet by going to Settings > Help & Support.
